packages/flutter_app_badger/example/integration_test/flutter_app_badger_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Since you are the original author of this code.

```suggestion
// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
```